Repeatmodeler test on osx

Juke34 opened the pull request:
**Replace this sentence with a detailed description of your pull request**

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

Juke34 commented:
Problems are 'recon' and 'repeatscout' not existing yet for osx

jebrosen commented:
'recon' and 'repeatscout' have had `skip: True # [not linux]` since they were first added and I'm not sure why that is the case.

Juke34 commented:
I have updated the recipe of RepeatScout to the latest version, I have then realised that it compiles well on OSX, so I have removed `skip: True # [not linux]` see #20098

Juke34 commented:
I have a working recipe for RepeatModeler2. I will upload it soon. Would like to fix this old recipe first (osx compatible). Working now on recon (URL has changed) see #20122

dpryan79 commented:
@bioconda-bot please fetch artifacts

dpryan79 commented:
@bioconda-bot please merge

Thanks @Juke34 !
